types/koa-session/koa-session-tests.ts

```diff
@@ -1,5 +1,5 @@
 import * as Koa from 'koa';
-import session = require('koa-session');
```

Comment:
> This syntax will make the resulting type any, which makes any tests in this file invalid.

this is not accurate. please see http://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.